### PR TITLE
Fixed new non-text calculations output a missing-label warning

### PR DESCRIPTION
**Pull request by gushil**
Closes #
#451 
#### Why is this the best possible solution? Were any other approaches considered?
The simplest one.
#### What are the regression risks?
None.
#### Does this change require updates to documentation? If so, please file an issue [here](https://github.com/XLSForm/xlsform.github.io) and include the link below.
None.
#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests_v1`
- [x] run `nosetests` and verified all tests pass
- [x] run `black pyxform` to format code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

**Comment by gushil** [comment]
@MartijnR I haven't added any test cases for this issue yet because I couldn't find a way to test warning yet. Do you have any suggestions about that?

Thanks

**Comment by MartijnR** [comment]
Thanks! I'll do some testing. Wrt to testing for warnings, see example here: https://github.com/XLSForm/pyxform/blob/master/pyxform/tests_v1/test_dynamic_default.py#L310-L316

**Comment by MartijnR** [comment]
I did not find issues with the functionality.

File: pyxform/xls2json.py [hunk 1]
```diff
@@ -883,6 +883,11 @@ def workbook_to_json(
             constants.LABEL not in row
             and row.get(constants.MEDIA) is None
             and question_type not in aliases.label_optional_types
+            and not row.get("bind", {}).get("calculate")
+            and not (
+                row.get("default")
+                and default_is_dynamic(row.get("default"), question_type)
```
Comment (MartijnR) [inline pyxform/xls2json.py:889]: Yes, excluding dynamic defaults as well! 👍 Would be good to add a separate test for that.

Comment (gushil) [reply pyxform/xls2json.py:889]: Ok, will add that.

**Comment by gushil** [comment]
@MartijnR I'm adding some tests.

Thanks.

**Pull request merged**